### ISSUE TITLE
Failed to install. Fatal: Not a git repository

### ISSUE THREAD

**Issue by euglevi** [wbthomason/packer.nvim#454]
I have Neovim 5.0, stable release in the form of an appimage on Ubuntu 21.04. I tried installing packer.nvim following the quickstart guide step by step.

My minimal plugins.lua has only nvim-tree.lua. I tried with other plugins too. Once I run PackerSync I always get the same error, even if by the packer_compiled.lua and by the list of commands it seems that the plugin is installed after all.

So, I really do not know what it is and if I should care.

Post the contents of ~/.cache/nvim/packer.nvim.log here

```
 ✗ Failed to install kyazdani42/nvim-tree.lua
  Errors:
    Clone in '/home/eugenio/.local/share/nvim/site/pack/packer/start/nvim-tree.lua' in corso...
    remote: Enumerating objects: 512, done.        
    remote: Counting objects:   0% (1/512)        
    remote: Counting objects:   1% (6/512)        
    remote: Counting objects:   2% (11/512)
    remote: Counting objects:   3% (16/512)        
    remote: Counting objects:   4% (21/512)        
    remote: Counting objects:   5% (26/512)        
    remote: Counting objects:   6% (31/512)        
    remote: Counting objects:   7% (36/512)        
    remote: Counting objects:   8% (41/512)        
    remote: Counting objects:   9% (47/512)        
    remote: Counting objects:  10% (52/512)        
    remote: Counting objects:  11% (57/512)        
    remote: Counting objects:  12% (62/512)        
    remote: Counting objects:  13% (67/512)        
    remote: Counting objects:  14% (72/512)        
    remote: Counting objects:  15% (77/512)        
    remote: Counting objects:  16% (82/512)        
    remote: Counting objects:  17% (88/512)        
    remote: Counting objects:  18% (93/512)        
    remote: Counting objects:  19% (98/512)        
    remote: Counting objects:  20% (103/512)        
    remote: Counting objects:  21% (108/512)        
    remote: Counting objects:  22% (113/512)        
    remote: Counting objects:  23% (118/512)        
    remote: Counting objects:  24% (123/512)        
    remote: Counting objects:  25% (128/512)        
    remote: Counting objects:  26% (134/512)        
    remote: Counting objects:  27% (139/512)        
    remote: Counting objects:  28% (144/512)        
    remote: Counting objects:  29% (149/512)        
    remote: Counting objects:  30% (154/512)        
    remote: Counting objects:  31% (159/512)        
    remote: Counting objects:  32% (164/512)        
    remote: Counting objects:  33% (169/512)        
    remote: Counting objects:  34% (175/512)        
    remote: Counting objects:  35% (180/512)        
    remote: Counting objects:  36% (185/512)        
    remote: Counting objects:  37% (190/512)        
    remote: Counting objects:  38% (195/512)        
    remote: Counting objects:  39% (200/512)        
    remote: Counting objects:  40% (205/512)        
    remote: Counting objects:  41% (210/512)        
    remote: Counting objects:  42% (216/512)        
    remote: Counting objects:  43% (221/512)        
    remote: Counting objects:  44% (226/512)        
    remote: Counting objects:  45% (231/512)        
    remote: Counting objects:  46% (236/512)        
    remote: Counting objects:  47% (241/512)        
    remote: Counting objects:  48% (246/512)        
    remote: Counting objects:  49% (251/512)        
    remote: Counting objects:  50% (256/512)        
    remote: Counting objects:  51% (262/512)        
    remote: Counting objects:  52% (267/512)        
    remote: Counting objects:  53% (272/512)        
    remote: Counting objects:  54% (277/512)        
    remote: Counting objects:  55% (282/512)        
    remote: Counting objects:  56% (287/512)        
    remote: Counting objects:  57% (292/512)        
    remote: Counting objects:  58% (297/512)        
    remote: Counting objects:  59% (303/512)        
    remote: Counting objects:  60% (308/512)        
    remote: Counting objects:  61% (313/512)        
    remote: Counting objects:  62% (318/512)        
    remote: Counting objects:  63% (323/512)        
    remote: Counting objects:  64% (328/512)        
    remote: Counting objects:  65% (333/512)        
    remote: Counting objects:  66% (338/512)        
    remote: Counting objects:  67% (344/512)        
    remote: Counting objects:  68% (349/512)        
    remote: Counting objects:  69% (354/512)        
    remote: Counting objects:  70% (359/512)        
    remote: Counting objects:  71% (364/512)        
    remote: Counting objects:  72% (369/512)        
    remote: Counting objects:  73% (374/512)        
    remote: Counting objects:  74% (379/512)        
    remote: Counting objects:  75% (384/512)        
    remote: Counting objects:  76% (390/512)        
    remote: Counting objects:  77% (395/512)        
    remote: Counting objects:  78% (400/512)        
    remote: Counting objects:  79% (405/512)        
    remote: Counting objects:  80% (410/512)        
    remote: Counting objects:  81% (415/512)        
    remote: Counting objects:  82% (420/512)        
    remote: Counting objects:  83% (425/512)        
    remote: Counting objects:  84% (431/512)        
    remote: Counting objects:  85% (436/512)        
    remote: Counting objects:  86% (441/512)        
    remote: Counting objects:  87% (446/512)        
    remote: Counting objects:  88% (451/512)        
    remote: Counting objects:  89% (456/512)        
    remote: Counting objects:  90% (461/512)        
    remote: Counting objects:  91% (466/512)        
    remote: Counting objects:  92% (472/512)        
    remote: Counting objects:  93% (477/512)        
    remote: Counting objects:  94% (482/512)        
    remote: Counting objects:  95% (487/512)        
    remote: Counting objects:  96% (492/512)        
    remote: Counting objects:  97% (497/512)        
    remote: Counting objects:  98% (502/512)        
    remote: Counting objects:  99% (507/512)        
    remote: Counting objects: 100% (512/512)        
    remote: Counting objects: 100% (512/512), done.        
    remote: Compressing objects:   0% (1/380)        
    remote: Compressing objects:   1% (4/380)        
    remote: Compressing objects:   2% (8/380)        
    remote: Compressing objects:   3% (12/380)        
    remote: Compressing objects:   4% (16/380)        
    remote: Compressing objects:   5% (19/380)        
    remote: Compressing objects:   6% (23/380)        
    remote: Compressing objects:   7% (27/380)        
    remote: Compressing objects:   8% (31/380)        
    remote: Compressing objects:   9% (35/380)        
    remote: Compressing objects:  10% (38/380)        
    remote: Compressing objects:  11% (42/380)        
    remote: Compressing objects:  12% (46/380)        
    remote: Compressing objects:  13% (50/380)        
    remote: Compressing objects:  14% (54/380)        
    remote: Compressing objects:  15% (57/380)        
    remote: Compressing objects:  16% (61/380)        
    remote: Compressing objects:  17% (65/380)        
    remote: Compressing objects:  18% (69/380)        
    remote: Compressing objects:  19% (73/380)        
    remote: Compressing objects:  20% (76/380)        
    remote: Compressing objects:  21% (80/380)        
    remote: Compressing objects:  22% (84/380)        
    remote: Compressing objects:  23% (88/380)        
    remote: Compressing objects:  24% (92/380)        
    remote: Compressing objects:  25% (95/380)        
    remote: Compressing objects:  26% (99/380)        
    remote: Compressing objects:  27% (103/380)        
    remote: Compressing objects:  28% (107/380)        
    remote: Compressing objects:  29% (111/380)        
    remote: Compressing objects:  30% (114/380)        
    remote: Compressing objects:  31% (118/380)        
    remote: Compressing objects:  32% (122/380)        
    remote: Compressing objects:  33% (126/380)        
    remote: Compressing objects:  34% (130/380)        
    remote: Compressing objects:  35% (133/380)        
    remote: Compressing objects:  36% (137/380)        
    remote: Compressing objects:  37% (141/380)        
    remote: Compressing objects:  38% (145/380)        
    remote: Compressing objects:  39% (149/380)        
    remote: Compressing objects:  40% (152/380)        
    remote: Compressing objects:  41% (156/380)        
    remote: Compressing objects:  42% (160/380)        
    remote: Compressing objects:  43% (164/380)        
    remote: Compressing objects:  44% (168/380)        
    remote: Compressing objects:  45% (171/380)        
    remote: Compressing objects:  46% (175/380)        
    remote: Compressing objects:  47% (179/380)        
    remote: Compressing objects:  48% (183/380)        
    remote: Compressing objects:  49% (187/380)        
    remote: Compressing objects:  50% (190/380)        
    remote: Compressing objects:  51% (194/380)
    remote: Compressing objects:  52% (198/380)        
remote: Compressing objects:  53% (202/380)
    remote: Compressing objects:  54% (206/380)        
remote: Compressing objects:  55% (209/380)        
remote: Compressing objects:  56% (213/380)        
remote: Compressing objects:  57% (217/380)        
remote: Compressing objects:  58% (221/380)        
remote: Compressing objects:  59% (225/380)        
remote: Compressing objects:  60% (228/380)        
remote: Compressing objects:  61% (232/380)        
remote: Compressing objects:  62% (236/380)        
remote: Compressing objects:  63% (240/380)        
remote: Compressing objects:  64% (244/380)        
remote: Compressing objects:  65% (247/380)        
remote: Compressing objects:  66% (251/380)        
remote: Compressing objects:  67% (255/380)        
remote: Compressing objects:  68% (259/380)        
remote: Compressing objects:  69% (263/380)
    remote: Compressing objects:  70% (266/380)        
remote: Compressing objects:  71% (270/380)        
remote: Compressing objects:  72% (274/380)        
remote: Compressing objects:  73% (278/380)        
remote: Compressing objects:  74% (282/380)        
remote: Compressing objects:  75% (285/380)        
remote: Compressing objects:  76% (289/380)        
remote: Compressing objects:  77% (293/380)        
remote: Compressing objects:  78% (297/380)        
remote: Compressing objects:  79% (301/380)        
remote: Compressing objects:  80% (304/380)        
remote: Compressing objects:  81% (308/380)        
remote: Compressing objects:  82% (312/380)        
remote: Compressing objects:  83% (316/380)        
remote: Compressing objects:  84% (320/380)        
remote: Compressing objects:  85% (323/380)        
remote: Compressing objects:  86% (327/380)        
remote: Compressing objects:  87% (331/380)        
remote: Compressing objects:  88% (335/380)        
remote: Compressing objects:  89% (339/380)        
remote: Compressing objects:  90% (342/380)        
remote: Compressing objects:  91% (346/380)        
remote: Compressing objects:  92% (350/380)        
remote: Compressing objects:  93% (354/380)        
remote: Compressing objects:  94% (358/380)        
remote: Compressing objects:  95% (361/380)        
remote: Compressing objects:  96% (365/380)        
remote: Compressing objects:  97% (369/380)        
remote: Compressing objects:  98% (373/380)        
remote: Compressing objects:  99% (377/380)        
remote: Compressing objects: 100% (380/380)        
remote: Compressing objects: 100% (380/380), done.
    Ricezione degli oggetti:   0% (1/512)
Ricezione degli oggetti:   1% (6/512)
Ricezione degli oggetti:   2% (11/512)
Ricezione degli oggetti:   3% (16/512)
Ricezione degli oggetti:   4% (21/512)
Ricezione degli oggetti:   5% (26/512)
    Ricezione degli oggetti:   6% (31/512)
    Ricezione degli oggetti:   7% (36/512)
    Ricezione degli oggetti:   8% (41/512)
    Ricezione degli oggetti:   9% (47/512)
    Ricezione degli oggetti:  10% (52/512)
Ricezione degli oggetti:  11% (57/512)
    Ricezione degli oggetti:  12% (62/512)
    Ricezione degli oggetti:  13% (67/512)
    Ricezione degli oggetti:  14% (72/512)
    Ricezione degli oggetti:  15% (77/512)
Ricezione degli oggetti:  16% (82/512)
    Ricezione degli oggetti:  17% (88/512)
Ricezione degli oggetti:  18% (93/512)
Ricezione degli oggetti:  19% (98/512)
    Ricezione degli oggetti:  20% (103/512)
Ricezione degli oggetti:  21% (108/512)
Ricezione degli oggetti:  22% (113/512)
    Ricezione degli oggetti:  23% (118/512)
Ricezione degli oggetti:  24% (123/512)
Ricezione degli oggetti:  25% (128/512)
Ricezione degli oggetti:  26% (134/512)
Ricezione degli oggetti:  27% (139/512)
    Ricezione degli oggetti:  28% (144/512)
Ricezione degli oggetti:  29% (149/512)
    Ricezione degli oggetti:  30% (154/512)
Ricezione degli oggetti:  31% (159/512)
Ricezione degli oggetti:  32% (164/512)
Ricezione degli oggetti:  33% (169/512)
    Ricezione degli oggetti:  34% (175/512)
Ricezione degli oggetti:  35% (180/512)
Ricezione degli oggetti:  36% (185/512)
Ricezione degli oggetti:  37% (190/512)
Ricezione degli oggetti:  38% (195/512)
Ricezione degli oggetti:  39% (200/512)
    Ricezione degli oggetti:  40% (205/512)
Ricezione degli oggetti:  41% (210/512)
    Ricezione degli oggetti:  42% (216/512)
Ricezione degli oggetti:  43% (221/512)
    Ricezione degli oggetti:  44% (226/512)
Ricezione degli oggetti:  45% (231/512)
    Ricezione degli oggetti:  46% (236/512)
Ricezione degli oggetti:  47% (241/512)
Ricezione degli oggetti:  48% (246/512)
Ricezione degli oggetti:  49% (251/512)
    Ricezione degli oggetti:  50% (256/512)
Ricezione degli oggetti:  51% (262/512)
Ricezione degli oggetti:  52% (267/512)
Ricezione degli oggetti:  53% (272/512)
    Ricezione degli oggetti:  54% (277/512)
Ricezione degli oggetti:  55% (282/512)
Ricezione degli oggetti:  56% (287/512)
Ricezione degli oggetti:  57% (292/512)
Ricezione degli oggetti:  58% (297/512)
    Ricezione degli oggetti:  59% (303/512)
    Ricezione degli oggetti:  60% (308/512)
Ricezione degli oggetti:  61% (313/512)
Ricezione degli oggetti:  62% (318/512)
Ricezione degli oggetti:  63% (323/512)
Ricezione degli oggetti:  64% (328/512)
Ricezione degli oggetti:  65% (333/512)
Ricezione degli oggetti:  66% (338/512)
Ricezione degli oggetti:  67% (344/512)
Ricezione degli oggetti:  68% (349/512)
Ricezione degli oggetti:  69% (354/512)
Ricezione degli oggetti:  70% (359/512)
Ricezione degli oggetti:  71% (364/512)
    Ricezione degli oggetti:  72% (369/512)
Ricezione degli oggetti:  73% (374/512)
Ricezione degli oggetti:  74% (379/512)
Ricezione degli oggetti:  75% (384/512)
    Ricezione degli oggetti:  76% (390/512)
Ricezione degli oggetti:  77% (395/512)
Ricezione degli oggetti:  78% (400/512)
Ricezione degli oggetti:  79% (405/512)
Ricezione degli oggetti:  80% (410/512)
    Ricezione degli oggetti:  81% (415/512)
    Ricezione degli oggetti:  82% (420/512)
    Ricezione degli oggetti:  83% (425/512)
Ricezione degli oggetti:  84% (431/512)
Ricezione degli oggetti:  85% (436/512)
Ricezione degli oggetti:  86% (441/512)
Ricezione degli oggetti:  87% (446/512)
Ricezione degli oggetti:  88% (451/512)
    Ricezione degli oggetti:  89% (456/512)
Ricezione degli oggetti:  90% (461/512)
Ricezione degli oggetti:  91% (466/512)
Ricezione degli oggetti:  92% (472/512)
    Ricezione degli oggetti:  93% (477/512)
    Ricezione degli oggetti:  94% (482/512)
    remote: Total 512 (delta 289), reused 262 (delta 112), pack-reused 0        
    Ricezione degli oggetti:  95% (487/512)
    Ricezione degli oggetti:  96% (492/512)
    Ricezione degli oggetti:  97% (497/512)
    Ricezione degli oggetti:  98% (502/512)
    Ricezione degli oggetti:  99% (507/512)
    Ricezione degli oggetti: 100% (512/512)
    Ricezione degli oggetti: 100% (512/512), 1.85 MiB | 4.89 MiB/s, fatto.
    Risoluzione dei delta:   0% (0/289)
Risoluzione dei delta:   1% (3/289)
Risoluzione dei delta:   2% (6/289)
Risoluzione dei delta:   3% (9/289)
Risoluzione dei delta:   4% (12/289)
Risoluzione dei delta:   5% (15/289)
Risoluzione dei delta:   6% (18/289)
Risoluzione dei delta:   7% (21/289)
Risoluzione dei delta:   8% (24/289)
Risoluzione dei delta:   9% (27/289)
Risoluzione dei delta:  10% (29/289)
Risoluzione dei delta:  11% (32/289)
Risoluzione dei delta:  12% (35/289)
Risoluzione dei delta:  13% (38/289)
Risoluzione dei delta:  14% (41/289)
Risoluzione dei delta:  15% (44/289)
Risoluzione dei delta:  16% (47/289)
Risoluzione dei delta:  17% (50/289)
Risoluzione dei delta:  18% (53/289)
Risoluzione dei delta:  19% (55/289)
Risoluzione dei delta:  20% (58/289)
Risoluzione dei delta:  21% (61/289)
Risoluzione dei delta:  22% (64/289)
Risoluzione dei delta:  23% (67/289)
Risoluzione dei delta:  24% (70/289)
Risoluzione dei delta:  25% (73/289)
Risoluzione dei delta:  26% (76/289)
Risoluzione dei delta:  27% (79/289)
Risoluzione dei delta:  28% (81/289)
Risoluzione dei delta:  29% (84/289)
Risoluzione dei delta:  30% (87/289)
Risoluzione dei delta:  31% (90/289)
Risoluzione dei delta:  32% (93/289)
Risoluzione dei delta:  33% (96/289)
Risoluzione dei delta:  34% (99/289)
Risoluzione dei delta:  35% (102/289)
Risoluzione dei delta:  36% (105/289)
Risoluzione dei delta:  37% (107/289)
Risoluzione dei delta:  38% (110/289)
Risoluzione dei delta:  39% (113/289)
    Risoluzione dei delta:  40% (116/289)
Risoluzione dei delta:  41% (119/289)
Risoluzione dei delta:  42% (122/289)
Risoluzione dei delta:  43% (125/289)
    Risoluzione dei delta:  44% (128/289)
Risoluzione dei delta:  45% (131/289)
Risoluzione dei delta:  46% (133/289)
    Risoluzione dei delta:  47% (136/289)
    Risoluzione dei delta:  48% (139/289)
    Risoluzione dei delta:  49% (142/289)
Risoluzione dei delta:  50% (145/289)
Risoluzione dei delta:  51% (148/289)
    Risoluzione dei delta:  52% (151/289)
Risoluzione dei delta:  53% (154/289)
Risoluzione dei delta:  54% (157/289)
Risoluzione dei delta:  55% (159/289)
Risoluzione dei delta:  56% (162/289)
Risoluzione dei delta:  57% (165/289)
Risoluzione dei delta:  58% (168/289)
Risoluzione dei delta:  59% (171/289)
    Risoluzione dei delta:  60% (174/289)
    Risoluzione dei delta:  61% (177/289)
    Risoluzione dei delta:  62% (180/289)
    Risoluzione dei delta:  63% (183/289)
    Risoluzione dei delta:  64% (185/289)
Risoluzione dei delta:  65% (188/289)
Risoluzione dei delta:  66% (191/289)
Risoluzione dei delta:  67% (194/289)
    Risoluzione dei delta:  68% (197/289)
Risoluzione dei delta:  69% (200/289)
Risoluzione dei delta:  70% (203/289)
Risoluzione dei delta:  71% (206/289)
Risoluzione dei delta:  72% (209/289)
Risoluzione dei delta:  73% (211/289)
Risoluzione dei delta:  74% (214/289)
Risoluzione dei delta:  75% (217/289)
Risoluzione dei delta:  76% (220/289)
Risoluzione dei delta:  77% (223/289)
    Risoluzione dei delta:  78% (226/289)
Risoluzione dei delta:  79% (229/289)
Risoluzione dei delta:  80% (232/289)
    Risoluzione dei delta:  81% (235/289)
Risoluzione dei delta:  82% (237/289)
    Risoluzione dei delta:  83% (240/289)
    Risoluzione dei delta:  84% (243/289)
Risoluzione dei delta:  85% (246/289)
Risoluzione dei delta:  86% (249/289)
Risoluzione dei delta:  87% (252/289)
Risoluzione dei delta:  88% (255/289)
Risoluzione dei delta:  89% (258/289)
    Risoluzione dei delta:  90% (261/289)
Risoluzione dei delta:  91% (263/289)
Risoluzione dei delta:  92% (266/289)
    Risoluzione dei delta:  93% (269/289)
Risoluzione dei delta:  94% (272/289)
Risoluzione dei delta:  95% (275/289)
Risoluzione dei delta:  96% (278/289)
Risoluzione dei delta:  97% (281/289)
    Risoluzione dei delta:  98% (284/289)
    Risoluzione dei delta:  99% (287/289)
Risoluzione dei delta: 100% (289/289)
Risoluzione dei delta: 100% (289/289), fatto.
    fatal: .git non è un repository Git (né lo è alcuna delle directory genitrici)

```

```

-- Automatically generated packer.nvim plugin loader code

if vim.api.nvim_call_function('has', {'nvim-0.5'}) ~= 1 then
  vim.api.nvim_command('echohl WarningMsg | echom "Invalid Neovim version for packer.nvim! | echohl None"')
  return
end

vim.api.nvim_command('packadd packer.nvim')

local no_errors = pcall(function()

  local time
  local profile_info
  local should_profile = false
  if should_profile then
    local hrtime = vim.loop.hrtime
    profile_info = {}
    time = function(chunk, start)
      if start then
        profile_info[chunk] = hrtime()
      else
        profile_info[chunk] = (hrtime() - profile_info[chunk]) / 1e6
      end
    end
  else
    time = function(chunk, start) end
  end
  
local function save_profiles(threshold)
  local sorted_times = {}
  for chunk_name, time_taken in pairs(profile_info) do
    sorted_times[#sorted_times + 1] = {chunk_name, time_taken}
  end
  table.sort(sorted_times, function(a, b) return a[2] > b[2] end)
  local results = {}
  for i, elem in ipairs(sorted_times) do
    if not threshold or threshold and elem[2] > threshold then
      results[i] = elem[1] .. ' took ' .. elem[2] .. 'ms'
    end
  end

  _G._packer = _G._packer or {}
  _G._packer.profile_output = results
end

time([[Luarocks path setup]], true)
local package_path_str = "/home/eugenio/.cache/nvim/packer_hererocks/2.1.0-beta3/share/lua/5.1/?.lua;/home/eugenio/.cache/nvim/packer_hererocks/2.1.0-beta3/share/lua/5.1/?/init.lua;/home/eugenio/.cache/nvim/packer_hererocks/2.1.0-beta3/lib/luarocks/rocks-5.1/?.lua;/home/eugenio/.cache/nvim/packer_hererocks/2.1.0-beta3/lib/luarocks/rocks-5.1/?/init.lua"
local install_cpath_pattern = "/home/eugenio/.cache/nvim/packer_hererocks/2.1.0-beta3/lib/lua/5.1/?.so"
if not string.find(package.path, package_path_str, 1, true) then
  package.path = package.path .. ';' .. package_path_str
end

if not string.find(package.cpath, install_cpath_pattern, 1, true) then
  package.cpath = package.cpath .. ';' .. install_cpath_pattern
end

time([[Luarocks path setup]], false)
time([[try_loadstring definition]], true)
local function try_loadstring(s, component, name)
  local success, result = pcall(loadstring(s))
  if not success then
    vim.schedule(function()
      vim.api.nvim_notify('packer.nvim: Error running ' .. component .. ' for ' .. name .. ': ' .. result, vim.log.levels.ERROR, {})
    end)
  end
  return result
end

time([[try_loadstring definition]], false)
time([[Defining packer_plugins]], true)
_G.packer_plugins = {
  ["nvim-tree.lua"] = {
    loaded = true,
    path = "/home/eugenio/.local/share/nvim/site/pack/packer/start/nvim-tree.lua"
  },
  ["packer.nvim"] = {
    loaded = true,
    path = "/home/eugenio/.local/share/nvim/site/pack/packer/start/packer.nvim"
  }
}

time([[Defining packer_plugins]], false)
if should_profile then save_profiles() end

end)

if not no_errors then
  vim.api.nvim_command('echohl ErrorMsg | echom "Error in packer_compiled: ".v:exception | echom "Please check your config for correctness" | echohl None')
end

```

**Comment by akinsho** [comment]
@euglevi this *might* relate to the fix @nanotee made over in #453, I'm just guessing since the exact git error is in italian I'm guessing and so I'm not sure what exactly is going wrong, but there was a PR about a day ago to change how git handles the target directory that might be causing this issue for you since this is a new installation.

I terms of a fix you can either wait for that PR to be merged or you can go to your packer repository locally and checkout either the branch and help us test if that fixes your issue or roll back the HEAD for packer a few commits and add the `frozen` (maybe it's freeze key to your use packer line)

**Comment by euglevi** [comment]
Sorry about the Italian in the report. But you can guess the error easily by the title of the thread: it just does not locate the git repository. I tried the PR by @nanotee and it seems to solve the issue. I can confirm that by my test the installation procedure now seems to be working fine.

**Comment by akinsho** [comment]
This should be fixed by #453 let us know if you're still seeing this issue despite the fix @euglevi
